src/Filesystem/Stream.php

```diff
@@ -0,0 +1,16 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Tempest\Filesystem;
+
+interface Stream
```

Comment:
Is a stream used somewhere? Or is it just an alternative provided by the framework?

Reply:
A stream interface is returned by `createStream` on the Filesystem driver so you could create a file stream that is filesystem agnostic.

Reply:
Oh, I missed that, I think that's great :)